proposals/PROPOSALS.md

```diff
@@ -0,0 +1,142 @@
+
+### Formal evaluation (up to 5 iterations)
+
+While the majority of general technical commentary and feedback will occur in the proposals repo, it's hard to tell when a particular proposal is finalized. This what we call "Formal Evaluation". 
+
+Formal Evaluation of a proposal is done in iterations. The maximum number of iterations is five. These iterations take place in the HF Tech Track meetings and are usually monthly. However, they can last longer, in which case the author has more time to implement all the required changes. 
+
+The HF Technical Track decides the duration of the next iteration depending upon the feedback and complexity of the HIP. Consequently, authors have more time to prepare all the changes. If they finish their revision before the scheduled iteration, an HF Technical Track member will reschedule it for the next available meeting.
+
+During every iteration, the HF Technical Track reviews the changes (updated design document, progress with the implementation, etc) to the proposal. Based on the feedback, the HIP is either:
+
+1. **Accepted**, in which case the HF Technical Track will propose a release date.
+2. **Rejected**, in which case the HIP is closed and no longer evaluated in the future.
+3. **Postponed**, in which case the HF Technical Track sets aside the HIP under some conditions. When those conditions are met, the HIP can be resubmitted.
+4. **Under revision**, in which case the author needs to continue the formal evaluation and address all the HF Technical Track feedback. Thus, the follow-up discussion is scheduled for the next iteration.
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+When the author and the HF Technical Track members agree on the final document, it is formally accepted: assigned a number, and scheduled for a final formal presentation.
+
+### Formal presentation (up to 1 month)
+
+During the next available Technical Track meeting, the author of the HIP presents the proposal to the audience. 
+
+### Merging the proposal
+
+If the HIP is accepted, the HF Technical Track will propose a release date, and the proposal will be merged directly by the HF Technical Track members.
+
+## Structure of the process
+
+The HIP process involves the following parties:
+
+1. The HIP Authors
+2. Relevant stakeholders affected by the change
+3. The HF Technical Track Leads
```

Comment:
Do we have a list of these somewhere? We don't mention any _Leads_ specifically in this document.

Reply:
Leads should be members - thanks I missed one reference

Reply:
Done